<launch>

  <arg name="model" default="$(find homework1_jrcb)/urdf/model.urdf"/>
  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
  
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find homework1_jrcb)/launch/urdf.rviz"  />

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
      <param name="use_gui" value="False"/>
  </node>


</launch>
